app/src/main/res/values/styles.xml

```diff
@@ -32,6 +32,13 @@
         <item name="android:layout_margin">@dimen/title_margin_minor_size</item>
     </style>
 
+    <style name="FloatingButtons">
```

Comment:
renamed all the styes, not representt the place witch are using

Reply:
Done thx